### DISCUSSION TITLE
additionally build ubuntu-based docker containers

### DISCUSSION THREAD

**Discussion by cany748** [directus/directus#23951] [Feature Requests]
### Describe the Improvement

I'd like to suggest additionally building docker images based on the “node:18” image instead of “node:18-alpine”. it won't be a change for current users if you use a different tag, such as “11.1.2-ubuntu”.
it will make life very much easier for many people, as it is not always convenient to use alpine-based images. at the same time, it is low maintenance, you only need to change a few lines in the workflows file.

I'm ready to make a pr!

**Comment by ryanleung96** [comment]
@cany748 Great idea, I would suggest adding supports to other images as well such as RHEL Ubi8. I built the image from source code with modifications on the Dockerfile.

**Comment by br41nslug** [comment]
Moving this to a feature request to gauge community interest in it 😄
